main.go

```diff
@@ -67,13 +71,21 @@ func main() {
 		Port:                   9443,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "f0fb677d.authorino.3scale.net",
+		LeaderElectionID:       "aac3a15d.kuadrant.3scale.net",
 	})
 	if err != nil {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
 
+	if err = (&controllers.AuthorinoReconciler{
+		Client: mgr.GetClient(),
+		Log:    ctrl.Log.WithName("authorino-operator").WithName("controller").WithName("Authorino"),
```

Comment:
We could use github.com/kuadrant/authorino/pkg/common/log.

Reply:
Yup, I'll create an issue to add the same log package that Authorino uses